ramls/feefine-reports.raml

```diff
@@ -97,3 +102,27 @@ traits:
             body:
               text/plain:
                 example: "Internal server error"
+  /financial-transactions-detail:
+      post:
+        is: [validate]
+        description: "Return data for a financial transactions detail report"
+        body:
+          application/json:
+            schema: financialTransactionsDetailReportRequest
+            example: !include examples/financialTransactionsDetailReportRequest.sample
+        responses:
+          200:
+            body:
+              application/json:
+                schema: financialTransactionsDetailReport
+                example: !include examples/financialTransactionsDetailReport.sample
+          422:
```

Comment:
I don't think we need to declare 422 explicitly, that `[validate]` trait you added should do the trick.